pkg/zcache/combined_cache.go

```diff
@@ -64,10 +64,17 @@ func (c *combinedCache) Get(ctx context.Context, key string, data interface{}) e
 		}
 
 		c.logger.Sugar().Debugf("set value found on remote cache in the local cache, key: [%s]", key)
+		ttl, ttlErr := c.remoteCache.TTL(ctx, key)
+		if ttlErr != nil {
+			c.logger.Sugar().Errorf("error getting TTL for key [%s] from remote cache, err: %s", key, ttlErr)
+			ttl = c.ttl
```

Comment:
We could remove now the global ttl config if we now get the TTL from remote. What do you think?